analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/RequestsWithExcessiveLength/Values/WrongPath/Web.config

```diff
@@ -0,0 +1,12 @@
+<configuration>
```

Comment:
please add comments to explain that `httpRuntime should be inside system.web` and `requestLimits should be inside requestFiltering` to help the reader quickly understand the test case

also, I would change `WrongPath` to something else like `InvalidConfig`  because WrongPath can refer to the FileSystem and would introduce confusion to the reader.